kgforge/specializations/models/rdf/rdf_model_service_from_store.py

```diff
@@ -91,22 +91,24 @@ def _build_shapes_map(self) -> Dict:
                     }}
                 }}
             }} ORDER BY ?type"""
+
         # make sure to get all types
         limit = 100
         offset = 0
         count = limit
-        class_to_shapes = dict()
-        shape_resource = dict()
+        class_being_shaped_id_to_shape_uri = dict()
+        schema_to_resource: Dict[URIRef, URIRef] = dict()
+
         while count == limit:
             resources = self.context_store.sparql(query, debug=False, limit=limit, offset=offset)
             for r in resources:
                 shape_uri = URIRef(r.shape)
-                class_to_shapes[r.type] = shape_uri
-                shape_resource[shape_uri] = URIRef(r.resource_id)
+                class_being_shaped_id_to_shape_uri[r.type] = shape_uri
```

Comment:
Looks like a very complicated name => class_being_shaped_id_to_shape_uri

Reply:
I've renamed it back to class_to_shape. But it was supposed to be:
key: id of the class that is the target of the shape
value: uri of the shape

Initially class_to_shape wasn't sufficient for me to understand what was in it so I made it a bit more verbose